index.html

```diff
@@ -23,32 +23,52 @@
       rel="stylesheet"
       href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
     />
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+   
   </head>
 
   <body>
     <header>
-      <nav class="mobile-version">
+      <nav class="mobile-version" id="mobile-nav">
         <div class="menu">
           <div class="shape">
             <div id="triangle"></div>
             <div id="diamond"></div>
           </div>
           <div>
-            <img id="menuimg" src="./assets/img/menu.png" alt="square" />
+            <button class="hamburger" id="menuimg" >
+              <i class="fa-sharp fa-solid fa-bars  fa-3x " id="hamburgericon"></i>
+            </button>
           </div>
         </div>
       </nav>
-      <nav class="desktop-version">
+      <div id="myNav" class="overlay">
+        <div class="mobile-menu-navbar">
+        <img src="./assets/img/navbar-mobile.png" alt="project1" id="mobile-menu-img" />
+        <a href="javascript:void(0)" class="closeicon" ><i class="closeIcon material-icons  fa-3x" id="close-icon">close</i></a>
+        </div>
+        <div class="overlay-content">
+          <a href="#intro" onclick="closeNav()">Hello</a>
+          <a href="#projects" onclick="closeNav()">Portfolio</a>
+          <a href="#about" onclick="closeNav()">About</a>
+          <a href="#form" onclick="closeNav()">Contact</a>
```

Comment:
- [Optional] The correct page was not displayed after clicking on this link. Please let's consider fixing it.